### DISCUSSION TITLE
SAML SSO with Okta Fails

### DISCUSSION THREAD

**Discussion by nikkisaildrone** [GitbookIO/community#588] [0. General bugs and questions]
We have a Pro Plan with Gitbook and our admins for Gitbook and our Okta org have followed the steps here to set up SAML SSO with our org's Okta account: https://docs.gitbook.com/account-management/sso-and-saml/saml#creating-end-user-account 

I have confirmed that I have been added to our Okta org's Gitbook app. I should be able to sign on to the site with Okta, yet every time I attempt to do so I hit this error:
![Screenshot 2024-04-19 at 1 23 33 PM](https://github.com/GitbookIO/community/assets/85899726/3240456e-c28c-461a-9fc8-2861ef6664ae)

It looks like Gitbook is consistently trying to route through Google SSO for login despite our admins having set up Gitbook to use Okta SSO. Is this a bug? Anything we can do to rectify this?

**Comment by aikoii** [comment]
Hey @nikkisaildrone, there could be a few reasons for this SSO issue – maybe the 'audience restriction' URL in Okta isn't correct, or you didn't copy the certificate properly (it should start and end with '---BEGIN CERTIFICATE---' and '---END CERTIFICATE---'). To get to the bottom of this, we'd need a closer look at your SAML setup. Could you send us the details at support@gitbook.com?

**Reply by fernando-mendoza1050** [reply]
Hey @aikoii we've been in contact with support@gitbook.com for about 2 weeks now. I've been following their steps to troubleshoot this but still no luck.

**Reply by aikoii** [reply]
I see! It looks like my colleague John has been keeping you updated on your case. Apologies for the delay; it took us longer than anticipated to address your issue. 🙏
